paasta_tools/cli/cmds/spark_run.py

```diff
@@ -639,7 +522,7 @@ def get_spark_app_name(
     if spark_app_name is None:
         spark_app_name = "paasta_spark_run"
 
-    spark_app_name += "_{}_{}".format(get_username(), spark_ui_port)
+    spark_app_name += f"_{get_username()}"
```

Comment:
Will this break https://github.com/Yelp/paasta/blob/master/paasta_tools/check_spark_jobs.py#L169 ? If so, can you update that, too?

Reply:
the spark_app_name will be passing to the service_configuration_lib and append ui_port and time
https://github.com/Yelp/service_configuration_lib/blob/master/service_configuration_lib/spark_config.py#L521

namely, the original code would generate 
`<spark_app_name>_<user>_<port>`
the new code is having the name 
`<spark_app_name>_<user>_<port>_<time>`
(appending `time` so that it matches the jupyterhub use case)  

Thanks for finding this out, I will fix that script!